includes/Ajax.php

```
@@ -86,7 +86,7 @@ public function admin_ajax() {
 	 * Get all Modules.
 	 */
 	private function get_all_modules() {
-		$modules = Modules::instance();
+		$modules = new ModuleManager();
 
 		wp_send_json( $modules->list_all_modules() );
 	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_all_modules'. (Unused Code Rules)

(UnusedPrivateMethod)